packages/vite-node/src/client.ts

```diff
@@ -269,6 +271,10 @@ export class ViteNodeRunner {
 
     const request = async (dep: string) => {
       const [id, depFsPath] = await this.resolveUrl(`${dep}`, fsPath)
+      const depMod = this.moduleCache.getByModuleId(depFsPath)
+      ;(depMod.importers ??= new Set()).add(moduleId)
+      ;(mod.imports ??= new Set()).add(depFsPath)
```

Comment:
Can you simplify this to looks cleaner?

```ts       
depMod.importers ??= new Set()
depMod.importers.add(moduleId)

mod.imports ??= new Set()
mod.imports.add(depFsPath)
```

We can also move this logic to `ModuleCache` with a nice name:

```ts
addImporter(mod, id) {
  mod ??= new Set()
  mod.importers.add(id)
}
addImport(mod, id) {
	mod.imports ??= new Set()
	mod.imports.add(id)
}
```